data/postgres.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @title Connect to Postgres
+ * @difficulty intermediate
+ * @tags cli, deploy
+ * @run --allow-net --allow-env <url>
+ * @resource {https://deno-postgres.com/} Deno Postgres docs
+ * @resource {https://deno.land/x/postgres} Deno Postgres on deno.land/x
+ *
+ * Using the Deno Postgres client, you can connect to a Postgres database
+ * running anywhere.
+ */
+
+// Import the Client constructor from deno.land/x
+import { Client } from "https://deno.land/x/postgres@v0.17.0/mod.ts";
```

Comment:
This kind of goes against the README's description of only using Deno built-ins and the std. I think that's okay, but we need to update the README.

Reply:
Rereading the README, it looks like there's specifically a line for exceptions. I guess exceptions should be make for database drivers because there are none in STD.